source/nodejs/adaptivecards-controls/src/inputwithpopup.ts

```diff
@@ -189,7 +189,7 @@ export abstract class InputWithPopup<TPopupControl extends PopupControl, TValue>
     private updateLabel() {
         if (this._labelElement) {
             if (this._value) {
-                this._labelElement.innerHTML = this.getValueAsString();
+                Utils.setInnerHtml(this._labelElement, this.getValueAsString());
```

Comment:
looks like this use of `innerHTML` was overzealous in the first place.

```suggestion
                this._labelElement.innerText = this.getValueAsString()
```

Reply:
I didn't get it. Need to fix this issue

![image](https://user-images.githubusercontent.com/6290692/189410500-ab94889b-f8b2-494f-829c-64041d36ce58.png)